src/ObjectExporter.php

```diff
@@ -0,0 +1,17 @@
+<?php declare(strict_types=1);
+/*
+ * This file is part of sebastian/exporter.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace SebastianBergmann\Exporter;
+
+interface ObjectExporter
```

Comment:
should a `ObjectExporter` have access to the `exporter` instance, so it can delegate export of nested properties, for which type a separate `ObjectExporter` exist?

Reply:
I think it should. If there is an object I need to export it will still have a lot of scalar values an arrays and it would be best if the default exporter is used there (or the chain)

Reply:
Implemented.